db/seeds.rb

```diff
@@ -1,14 +1,17 @@
 require 'faker'
 
+user = FactoryBot.create(:user, role: :research_group)
+
 6.times do |n|
 	title = Faker::Lorem.sentences(number: 1)
 	body = Faker::Lorem.paragraph
-	author = Faker::Name.name
-
+  author = Faker::Name.name
```

Comment:
Please fix formatting

Reply:
Code is formatted in code base, don't know why it shows like this here.

Reply:
Make sure to check your settings for that specific file. Are they set for 2 spaces?